src/website/app/demos/Link/bestPractices.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Link from '../../../../Link';
+
+export default [
+  {
+    type: 'do',
+    title: 'use Links to navigate',
+    example: (
+      <p>
+        <Link href="https://mineral-ui.com">Lorem ipsum</Link> dolor sit amet,
+        consectetur adipiscing elit. Donec mattis pretium massa. Aliquam erat
+        volutpat. Nulla facilisi. Donec vulputate interdum sollicitudin.
+      </p>
+    ),
+    description:
+      'Links should be used inline to navigate to other content or pages.'
```

Comment:
They can be used inline or on their own, e.g., in a navigation menu.

Reply:
ah yes.